Dockerfile

```diff
@@ -44,7 +44,7 @@ ENV cwms.dataapi.access.providers "KeyAccessManager,OpenID"
 ENV cwms.dataapi.access.openid.wellKnownUrl "https://identity-test.cwbi.us/auth/realms/cwbi/.well-known/openid-configuration"
 ENV cwms.dataapi.access.openid.issuer "https://identity-test.cwbi.us/auth/realms/cwbi"
 ENV cwms.dataapi.access.openid.timeout "604800"
-ENV cwms.dataapi.access.openid.altAuthUrl "https://identityc-test.cwbi.us/auth/realms/cwbi"
+#ENV cwms.dataapi.access.openid.altAuthUrl "https://identityc-test.cwbi.us/auth/realms/cwbi"
```

Comment:
I see its commented out but was identityc a typo?  why the c char?

Reply:
"CAC". without the C you can get to the .well-known, with the c you get the mtls version.

Not that this mechanism actually worked, unfortunately.